### PR TITLE
Add more JavaScript quiz questions to Developer_Quiz_Site

### PR DESCRIPTION
**Pull request by amehi0index**
Checklist:
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

File: src/data/full-quiz.js [hunk 1]
```diff
@@ -1,4 +1,59 @@
 const fullQuiz = [
+  {
+    Question:
+      "Which method returns an HTMLCollection object that consists of all of the elements that have the same class name?",
+    Answer: "getElementsByClassName()",
+    Distractor1: "querySelector()",
+    Distractor2: "getElementsByName()",
+    Distractor3: "getElementsByTagName()",
+    Explanation:
+    "The getElementsByClassName method of Document interface returns an array-like object of all child elements which have all of the given class name(s).",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:10]: I think the explanation can be simplified.

Proposed fix: The getElementsByClassName method returns an object containing all of the child elements with that given class name. 

I think simpler is always better. And if people want to learn more about this method they can always click on the MDN article for the official definition.

File: src/data/full-quiz.js [hunk 2]
```diff
@@ -1,4 +1,59 @@
...
+  },
+{
+    Question:
+      "What method is used to arrange array values into alphabetical and/or ascending order?",
+    Answer: "sort()",
+    Distractor1: "shift()",
+    Distractor2: "unshift()",
+    Distractor3: "from()",
+    Explanation:
+    "The sort() method sorts the elements of an array in place and returns the sorted array. The default sort order is ascending, built upon converting the elements into strings, then comparing their sequences of UTF-16 code units values",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:32]: I think we should shorten this explanation. 

Proposed fix: The sort() method sorts the elements of an array in place and returns the sorted array. The default sort order is ascending.

File: src/data/full-quiz.js [hunk 3]
```diff
@@ -1,4 +1,59 @@
...
+    Link: "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort",
+  },
+{
+    Question:
+      "What method is used to arrange array values into descending order?",
+    Answer: "reverse()",
+    Distractor1: "from()",
+    Distractor2: "unshift()",
+    Distractor3: "sort()",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:41]: I think we should change the answer for Distractor3 to be one of the more obviously wrong answers.
That way reverse() is the only logical choice especially for those newer to programming that might choose sort. 
We try to make these questions on the easier side so it is accessible to all levels. 
There will be future versions of the site, where people can choose quiz difficulty levels.  But for now, easier is better.

Proposed fix: join()

File: src/data/full-quiz.js [hunk 4]
```diff
@@ -1,4 +1,59 @@
...
+  },
+{
+    Question:
+      "The ____________ method returns the index position of the first occurrence of a value in a string.",
+    Answer: "indexOf()",
+    Distractor1: "getPosition()",
+    Distractor2: "valueOf()",
+    Distractor3: "charAt(0)",
+    Explanation:
+    "The indexOf() method returns the index within the calling String object of the first occurrence of the specified value, starting the search at fromIndex. Returns -1 if the value is not found.",
```
Comment (jdwilkin4) [inline src/data/full-quiz.js:54]: I think the explanation can be simplified.

Proposed fix: The indexOf() method returns the first index of the specified value in a string and will return  -1 if it is not present.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Hi @amehi0index ! 
I think your quiz questions are good.
I have just asked for a few small changes.

My overall comment would be to simplify the explanations.
Then if users want to explore more about a give topic, they can click on the mdn articles. 

Thanks! 👍

File: src/data/full-quiz.js (hunk 1 above)
Comment (amehi0index) [reply src/data/full-quiz.js]: Sounds good. Thank you for the feedback. :-)

File: src/data/full-quiz.js (hunk 2 above)
Comment (amehi0index) [reply src/data/full-quiz.js]: Sounds good. Thank you for the feedback. :-)

File: src/data/full-quiz.js (hunk 3 above)
Comment (amehi0index) [reply src/data/full-quiz.js]: Sounds good. Thank you for the feedback. :-)

File: src/data/full-quiz.js (hunk 4 above)
Comment (amehi0index) [reply src/data/full-quiz.js]: Sounds good. Thank you for the feedback. :-)

**Pull request merged**